### DISCUSSION TITLE
提个设想

### DISCUSSION THREAD

**Discussion by cxw620** [XIU2/CloudflareSpeedTest#333] [讨论]
能不能增加收集的功能, 让用户测速后上传测速地ip和测速结果(最优的两个)给大家共享使用呢? 国内用cf的网站也不少了. 这样就可DNS分省份解析到CF最佳的ip上面

**Comment by XIU2** [comment]
前段时间删库关闭的 better-cloudflare-ip 项目，就是因为这么干被质疑隐私而被攻击，最后项目作者心灰意冷，或者说感觉心累了，就删库了。。。
除了这个原因外，我也不会这么干的，因为折腾起来太麻烦，懒得搞。

**Comment by cxw620** [comment]
> 前段时间删库关闭的 better-cloudflare-ip 项目，就是因为这么干被质疑隐私而被攻击，最后项目作者心灰意冷，或者说感觉心累了，就删库了。。。 除了这个原因外，我也不会这么干的，因为折腾起来太麻烦，懒得搞。


了解了, 作者大大辛苦了
